packages/js-sdk/src/session/filesystem.ts

```diff
@@ -1,4 +1,5 @@
 import FilesystemWatcher from './filesystemWatcher'
+import { CallOpts } from './sessionConnection.ts'
```

Comment:
TS is (correctly) complaining about the `.ts` suffix

```suggestion
import { CallOpts } from './sessionConnection'
```

![image](https://github.com/e2b-dev/e2b/assets/697301/64e73b28-55bc-4b41-8e8f-4916d8bd30fa)